bigquery/dataset_size.js

```diff
@@ -0,0 +1,152 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var async = require('async');
+var google = require('googleapis');
+var tables = google.bigquery('v2').tables;
```

Comment:
Also - we need to figure out the best place to keep samples that use the api client vs the veneer.  We have a set of samples over in the other repository - when does it make sense to keep samples here vs the api client repo?